news/completer-py-api.rst

```diff
@@ -0,0 +1,27 @@
+**Added:**
+
+* index for history's sqlite-DB
+* support passing style from RichCompleter to PTK's Completer
+* ``xonsh.cli_utils`` to create cli from functions easily.
+* Python API for completer command with ``xonsh.completer`` module functions.
```

Comment:
Are all of these related to this PR or are some leftover from other PRs?

Reply:
1 & 2. required them when trying to create a history completer. 
3 & 4. cli-utils and completer are both related. 

If you want, I can create a separate PR. but these are contained within their respective commits.

Reply:
that's related enough for me.  thanks for clarifying!